tests/Integration/Database/DatabaseMySqlSchemaBuilderAlterTableWithEnumTest.php

```diff
@@ -58,7 +65,12 @@ public function testGetAllTablesAndColumnListing()
 
         $tableProperties = array_values((array) $tables[0]);
         $this->assertEquals(['users', 'BASE TABLE'], $tableProperties);
-        $this->assertEquals(['id', 'name', 'age', 'color'], Schema::getColumnListing('users'));
+
+        $columns = Schema::getColumnListing('users');
+
+        foreach (['id', 'name', 'age', 'color'] as $column) {
+            $this->assertContains($column, $columns);
+        }
```

Comment:
Really unhappy with this solution but there really doesn't seems like a better way to do this 😅

Reply:
you could sort the arrays and check they are same?

Reply:
actually, there's a function for that `assertEqualsCanonicalizing`